paddle/framework/data_transform.cc

```diff
@@ -76,12 +77,12 @@ void TransDataType(const platform::DeviceContext* ctx,
 
 void TransDataLayout(const platform::DeviceContext* ctx,
                      const KernelTypePair& kernel_pair, const Variable& in,
-                     Variable* out) {
+                     Variable* out, const std::vector<int>& axis) {
```

Comment:
Parameter order should be: inputs, then outputs.

Reply:
done